requirements.txt

```diff
@@ -1,6 +1,6 @@
-tensorflow==2.2
-keras==2.3.1
-protobuf==3.20.1
+tensorflow
```

Comment:
I would be inclined to keep these pinnings, but as a lower bound (e.g. tensorflow>=2.2), as the model may not work for versions below that (assuming that still solves the problem)